docs/scos/dev/feature-integration-guides/202200.0/company-account-feature-integration.md

```diff
@@ -0,0 +1,787 @@
+---
+title: Company Account feature integration
+description: The guide walks you through the process of installing Business on Behalf, Company Account Storage, and Company Account OAuth functionality into your project.
+last_updated: Jun 16, 2021
+template: feature-integration-guide-template
+originalLink: https://documentation.spryker.com/2021080/docs/company-account-feature-integration
+originalArticleId: f29e0e4a-c1f6-4812-ae43-a771909cb715
+redirect_from:
+  - /2021080/docs/company-account-feature-integration
```

Comment:
please ask TW if we need to use latest 202204 release

Reply:
TW answered that he will fix the links himself if needed.

I also added sections about Zed frontend and Zed translations.